src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const myDiv = document.createElement('div');
+  const myP = document.createElement('p');
+  const myH = document.createElement('h3');
```

Comment:
```suggestion
  const containerElement = document.createElement('div');
  const descriptionElement = document.createElement('p');
  const titleElement = document.createElement('h3');
```
такий неймінг мабуть буде більш зрозумілий)